src/components/text-input/index.md.njk

```diff
@@ -67,10 +67,20 @@ Fluid width inputs will resize with the viewport.
 
 ### Hint text
 
-Use hint for help that’s relevant to the majority of users, like how their information will be used, or where to find it.
+Use hint text for help that’s relevant to the majority of users, like how their information will be used, or where to find it.
 
 {{ example({group: "components", item: "text-input", example: "input-hint-text", html: true, nunjucks: true, open: false, size: "s"}) }}
 
+#### When not to use hint text
+
+Do not use hint text for long paragraphs and lists. Screen readers read out the entire text when users interact with the form element. This could frustrate users if the text is long.
```

Comment:
I stumbled a little on this bit – maybe it sounds a bit like someone might use hint text within long paragraphs or lists, rather than the other way round. So it could be more like 'Do not use long paragraphs or lists in hint text'?

Reply:
```suggestion
Do not use long paragraphs and lists in hint text. Screen readers read out the entire text when users interact with the form element. This could frustrate users if the text is long.
```